project/AppDependencies.scala

```diff
@@ -24,8 +24,6 @@ object AppDependencies {
   )
 
   val test: Seq[ModuleID] = Seq(
-//    "org.mockito"             %% "mockito-scala-scalatest"      % mockitoScalatestVersion,
-    "org.scalatestplus"       %% "mockito-4-11"                 % "3.2.18.0",
     "org.scalatestplus"       %% "scalacheck-1-17"              % scalatestPlusScalacheckVersion,
```

Comment:
On the basis of this, I think Line 27 and Line 28 can also be removed
https://github.com/hmrc/bootstrap-play/blob/main/project/LibDependencies.scala
Give this a go just to check if I'm correct, but if not don't worry about changing it.